nemo/core/classes/modelPT.py

```diff
@@ -371,8 +371,10 @@ def maybe_make_save_dir(path: 'pathlib.Path'):
                         'connector which supports model parallel mode, such as NLPSaveRestoreConnector in NLP. You '
                         'can also use a custom one.'
                     )
-            if app_state.data_parallel_rank == 0:
+            if is_global_rank_zero():
```

Comment:
Why change this?

Reply:
@MaximumEntropy had to change this to so save large parallel nemo files.  @MaximumEntropy can you comment?

Reply:
Having it the old way was failing for me since multiple model parallel ranks all think that the folder does not exist and all want to create the folder. But after one of them succeed, the others complain that the folder already exists.